scripts/Tools/case.build

```diff
@@ -94,20 +94,10 @@ def _main_func(description):
                     ts.set_status(phase_to_fail, TEST_FAIL_STATUS, comments="failed to initialize")
                 raise
 
-            append_status("case.testbuild starting ",
```

Comment:
CaseStatus handling for case.build needs to be in the library to ensure it's always done.